Add the rest of implemented GQLSTATUS error codes (#220)
neo4j/docs-status-codes: modules/ROOT/pages/errors/gql-errors.adoc

// Status description:: error: data exception - unexpected map entry. Invalid input. Unexpected key `{ $key }`, expected keys are `{ $listAllowedKeys }`.

=== 22NB0

> Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.

Reviewer: Not sure how you want to format this, but probably not with different quotes before and after WHERE. 

In the code it is ...evaluates to 'NULL'. Use `WHERE` syntax in combination with `IS NULL` ... so inconsistent (I think @renetapopova wanted to go through and fix these sort of inconsistencies), but not as broken as here in the docs.

Author: _invalid property based access control rule involving WHERE and IS NULL_ is subcondition. 
It seems you don't use any kind of quotes or backticks in subconditions (look at them in the spreadsheet or code.)
In another PR, I already asked the question about the Cypher clauses and expressions mentioned in the messages. If I remember properly, it was decided to  postpone this discussion.

Reviewer: Yes, we still have a trello card for making it more consistent. As long as the docs rendering is not broken, it should be good enough for now

> Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.

Author: ```suggestion
Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE` syntax in combination with `IS NULL` instead.
```

Status description:: error: syntax error or access rule violation - invalid `FOREACH`. Invalid use of `{ $clause }` inside 'FOREACH'.

> ??? === 42I02

Reviewer: What is the questions marks here for?

Author: forgot to remove them

> ??? === 42I02

Author: ```suggestion
=== 42I02
```

Status description:: error: syntax error or access rule violation - invalid use of `UNION` and `UNION ALL`. `UNION` and `UNION ALL` cannot be combined.

=== 42I45

Status description:: error: syntax error or access rule violation - invalid use of multiple path patterns. Multiple path patterns cannot be used in the same clause in combination with a selective path selector. `{ $action }`

=== 42I47

Status description:: error: syntax error or access rule violation - parser error. Parser Error: `{ $msg }`.

=== 42I48

Status description:: error: syntax error or access rule violation - invalid use of a subquery in MERGE. Subqueries are not allowed in a MERGE clause.

=== 42I50

Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ $input }..`. A `{ $tokenType }` name cannot be longer than `{ $maxTokenLength }`.

=== 42I51

> Status description:: info: invalid call signature. The procedure or function `{ %s }` must have the signature: `{ %s }`.

Reviewer: ```suggestion
Status description:: info: invalid call signature. The procedure or function `{ $procFun }` must have the signature: `{ $sig }`.
```

Status description:: error: syntax error or access rule violation - unsupported normal form. Unknown Normal Form: `{ $input }`.

=== 42N51

> Status description:: error: syntax error or access rule violation - invalid parameter. Invalid parameter `{ $param }`

Reviewer: ```suggestion
Status description:: error: syntax error or access rule violation - invalid parameter. Invalid parameter `{ $param }`.
```

Status description:: error: syntax error or access rule violation - cannot delete own user. Cannot delete the user record of the current user.

=== 42NA5

Status description:: info: accessing multiple graphs only supported on composite databases. Accessing multiple graphs in the same query is only supported on composite databases. Connect to a composite database with the desired constituents.

=== 42NA6

Status description:: info: invalid alias target. Aliases are not allowed to target composite databases.

=== 42NA7

> Status description:: info: referenced database not found. No database is corresponding to { $db }. Verify that the elementId is correct.

Reviewer: ```suggestion
Status description:: info: referenced database not found. No database is corresponding to `{ $db }`. Verify that the elementId is correct.
```
